blurb

```diff
@@ -650,24 +650,32 @@ Run unit tests.  Only works inside source repo, not when installed.
     print(tests_run, "tests passed.")
 
 
+def find_editor():
+    for var in 'GIT_EDITOR', 'EDITOR':
+        editor = os.environ.get(var)
+        if editor is not None:
+            return editor
+    if sys.platform == 'win32':
+        fallbacks = ['notepad.exe']
+    else:
+        fallbacks = ['/etc/alternatives/editor', 'nano']
+    for fallback in fallbacks:
+        if os.path.isabs(fallback):
+            found_path = fallback
+        else:
+            found_path = shutil.which(fallback)
+        if found_path and os.path.exists(found_path):
+            return found_path
+        error('Could not find an editor! Set the EDITOR environment variable.')
```

Comment:
Please outdent the error() call.  Right now it fails after the first alternative is tried.

(maybe my fault)